core/src/main/scala/org/apache/spark/Accumulable.scala

```diff
@@ -146,6 +198,32 @@ class Accumulable[R, T] private (
   def merge(term: R) { value_ = param.addInPlace(value_, term)}
 
   /**
+   * Merge in pending updates for ac consistent accumulators or merge accumulated values for
+   * regular accumulators.
```

Comment:
I'd add a comment that this is only gets called on the driver when merging Task results together.